src/media-groups.js

```diff
@@ -546,7 +546,7 @@ export const initialize = {
       }
 
       for (const variantLabel in mediaGroups[type][groupId]) {
-        if (mediaGroups[type][groupId][variantLabel].forced) {
+        if (!vhs.options_.useForcedSubtitles && mediaGroups[type][groupId][variantLabel].forced) {
```

Comment:
```suggestion
        if (!vhs.options().useForcedSubtitles && mediaGroups[type][groupId][variantLabel].forced) {
```

Reply:
@dzianis-dashkevich  thank you for taking the time to do the code review and the change proposal. 👍🏽 

I agree with the proposed change which seems more logical, that was my first approach. However, after looking at the code base I saw that instead of the method `options()` it was the property `options_` that was used. So I ended up using the property, to keep some consistency. Would you like me to make the change anyway?

Please see:
https://github.com/videojs/http-streaming/blob/93a2bfdb197ba552122784db430e816bec1f2130/src/media-groups.js#L650
https://github.com/videojs/http-streaming/blob/93a2bfdb197ba552122784db430e816bec1f2130/src/playlist-loader.js#L395

Reply:
Oh never mind then!
I guess it will be a separate effort to refactor all places in the future.
Thank you!